README.md

```diff
@@ -1,2 +1,66 @@
-# Bookstore
-Bookstore is a SPA where you can manage your books more effectively then a regular library. Built with React and Redux.
+# Math Magicians
+
```

Comment:
- [ ] The title here is different from what the project is about. Please fix the title of the project.